src/run_docker.py

```diff
@@ -174,8 +176,11 @@ def main():
     if OPERATION_MODE == OperationMode.Metrics:
         asyncio.run(run_metrics_fetcher_forever())
     elif OPERATION_MODE == OperationMode.Logs:
-        LogsFastCheck(logging_context, instance_metadata).execute()
-        run_logs(logging_context, instance_metadata)
+        asyncio.run(LogsFastCheck(logging_context, instance_metadata).execute())
+        processes = [multiprocessing.Process(target=run_logs_wrapper, args=(logging_context, instance_metadata, i))
+                     for i in range(PARALLEL_PROCESSES)]
+        [process.start() for process in processes]
+        [process.join() for process in processes]
```

Comment:
I think it's worth considering to call `process.close() `for each process, so it releases resources immediately. 
Also just a small question out of curiosity, was there any particular reason why you used lists for this iteration 😄